### ISSUE TITLE
PS2FlashightBrightness update

### ISSUE THREAD

**Issue by AeroWidescreen** [elishacloud/Silent-Hill-2-Enhancements#196]
@elishacloud 

This fixes a lighting bug that occured in the Heaven's Night hallway. This was apparently caused by PS2FlashightBrightness. Thanks to **Bigmanjapan**.

![LightingBug](https://i.imgur.com/VAHlyL4.gif)

You can simplify this even further if you want. I opted for legibility.

```
004FFCCC jmp 00000000
00000000 cmp dword ptr [RoomID], 0C
00000001 je 000000005

// if not Heaven's Night hallway
00000002 cmp eax,02
00000003 jg 004FFD7C
00000004 jmp 004FFCD5

// if Heaven's Night hallway
00000005 cmp eax,03
00000006 jg 004FFD7C
00000007 jmp 004FFCD5
```

---

This scales the inner glow of the flashlight for different resolutions. Thanks to **Polymega**.

Memory address `006C3254` is only referenced by three instructions, which are all exclusively used for the inner glow. It should be completely safe to change the value of this address.

`006C3254 41300000 // default value 11.0 float`

Formula:

Y Resolution / 480 = Result
Result * 11 = InnerGlowSize

**Comment by elishacloud** [comment]
Thanks @AeroWidescreen!  I will work on this after the New Year, since I probably won't have time before that.

**Comment by elishacloud** [comment]
Implemented both of these updates in the test build below. 

Here is a test build: [d3d8.zip](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/4080050/d3d8.zip)

**Comment by Polymega** [comment]
Confirmed working on all binaries, along with start00, start01, savebg, and savebg2. :heart:

**Comment by elishacloud** [comment]
Closing.  Implemented in 8b4e2fbe29e0d53abeafb66d6544605b074abde0
